src/AltinnCore/Runtime/Startup.cs

```diff
@@ -268,6 +268,17 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                        instanceId = @"^(\{{0,1}([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}\}{0,1})$",
                    });
 
+                routes.MapRoute(
+                   name: "runtimeRoute",
+                   template: "runtime/{org}/{service}",
```

Comment:
instead of creating a separate route, you can change the uieditroute to have the instanceid as an optional parameter like template: "runtime/{org}/{service}/{instanceId?}"